.github/workflows/continuous-benchmark.yaml

```diff
@@ -48,3 +48,23 @@ jobs:
       - name: Fail job if any of the benches failed
         if: steps.benches.outputs.failed == 'true'
         run: exit 1
+      - name: Send Notification
+        if: failure() || cancelled()
+        uses: slackapi/slack-github-action@v1.26.0
+        with:
+          payload: |
+            {
+              "text": "CI benchmarks run status: ${{ job.status }}",
+              "blocks": [
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "CI benchmarks failed or timed out.\nView the results <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|here>"
+                  }
+                }
+              ]
+            }
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.CRASHER_CHANNEL_WEBHOOK_URL }}
```

Comment:
Do not forget to create a new secret and provide it here.

Reply:
Done. Thanks!